ASFAchievementManager/ASF-Achievement-Manager.csproj

```diff
@@ -9,13 +9,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="JetBrains.Annotations" Version="*" IncludeAssets="compile" />
-    <PackageReference Include="SteamKit2" Version="*" IncludeAssets="compile" />
-    <PackageReference Include="System.Composition.AttributedModel" Version="*" />
+    <Reference Include="SteamKit2">
+      <HintPath>..\ArchiSteamFarm\ArchiSteamFarm\lib\SteamKit2.dll</HintPath>
```

Comment:
Since ASF itself is using the official version of SteamKit again we should either reference that one or remove this entry completely due to the transitivity of the dependency included in ASF (which will make sure we are using the same version).

Reply:
This is original code from @Ryzhehvost - so probably a disccusion for them?

Reply:
It works now, but at some point this will most likely lead to exceptions when executing a command in ASF (whenever something we call or inherit from steamkit has a breaking change between the two versions of SK2 that are loaded [the one we use and the one ASF uses]). Better just depend on the one that is already loaded. This was necessary because back then ASF was using a custom version of SK2 as well (referencing the .dll)

Reply:
Let's focus on the code, I can modify project file later.